barcampgb.org

```diff
@@ -0,0 +1 @@
+Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
```

Comment:
Done, Do I need to follow that up with a push or something?

On Wed, Jun 1, 2016 at 6:59 PM Chris Jaure <notifications@github.com> wrote:

> In barcampgb.org
> <https://github.com/BarCampGreenBay/barcampgb.org/pull/51#discussion_r65464294>
> :
>
> > @@ -0,0 +1 @@
> > +Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
>
> I cloned your repo and it appears as another directory called
> barcampgb.org. Seems like somehow this repo got included as a submodule,
> but the mapping isn't complete. You should be able to remove it by running rm
> -r barcampgb.org from command line.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BarCampGreenBay/barcampgb.org/pull/51/files/f253472d73b2761f72c09276d0a07c3b2aea4f8a#r65464294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIjw8arA4iFt_9B0stzNQYAj38JAmNCUks5qHhz2gaJpZM4IrILu>
> .
>
-- 
Sent from my iPhone